src/api/getArtifactsForBranchAndWorkflow.ts

```diff
@@ -61,8 +61,9 @@ export async function getArtifactsForBranchAndWorkflow(
       repo,
       // Below is typed incorrectly, it needs to be a string but typed as number
       workflow_id: (workflow_id as unknown) as number,
-      branch,
-      status: 'completed',
+      // XXX: GH broke filtering so we have to do it client-side for now
+      // branch,
+      // status: 'success',
```

Comment:
We'll want to keep this status change when we revert this change. This is more correct. (`status` is overloaded in the GH API, where it can filter either a workflow run's `status` *or* the `conclusion`).